Makefile

```diff
@@ -64,13 +64,15 @@ INCLUDEOPTS=$(shell gcc -E -x c++ - -v < /dev/null 2>&1 >/dev/null \
 	    | grep '^ /' | grep -v cc1plus | awk '{print "-I" $$1 " "}')
 PYINCLUDE=$(shell python -c \
 	  "import sysconfig;print(sysconfig.get_path('include'))")
+SYSPYINCLUDE=$(shell python -c \
+      "from __future__ import print_function; from distutils.sysconfig import get_python_inc; print(get_python_inc())")
```

Comment:
If you're using virtualenv, the first one will point to the virtualenv's include directory, while the second will point to the system's include directory (where the Python dev headers are).

```
(kenv)[standage@lappy khmer]$ python -c "from __future__ import print_function; import sysconfig; print(sysconfig.get_path('include'))"
/home/standage/Software/kenv/include/python3.4m
(kenv)[standage@lappy khmer]$ python -c "from __future__ import print_function; from distutils.sysconfig import get_python_inc; print(get_python_inc())"
/usr/include/python3.4m
```

Even when I am not in a virtualenv, these give different results.

```
[standage@lappy khmer]$ python -c "from __future__ import print_function; import sysconfig; print(sysconfig.get_path('include'))"
/usr/local/include/python2.7
[standage@lappy khmer]$ python -c "from __future__ import print_function; from distutils.sysconfig import get_python_inc; print(get_python_inc())"
/usr/include/python2.7
```

That first directory doesn't even exist on my system (Debian Jessie). I'll try to confirm whether we even need the old PYINCLUDE. It may be sufficient to overwrite the value instead of creating a new SYSPYINCLUDE value.